data/curated/traffic-flow/meta.yaml

```
@@ -0,0 +1,19 @@
+title: National Highways Traffic Flow
+article:
+  title: National Highways
+  url: https://webtris.nationalhighways.co.uk/
+data_source:
+  title: WebTRIS Traffic Flow API
+  url: https://webtris.nationalhighways.co.uk/api/swagger/ui/index
+images:
+# Please include at least one image, and up to three images
+- file: traffic-heatmap.png
+  alt: >
+    A heatmap of average speed of vehicles showing time of day on x-axis and day of week on y-axis for the TAME Site 30361338 during May 2021. Squares are colored from light peach to dark red with darker colors indicating higher speed. The speed scale ranges from 0 to 50 mph, and the chart shows some speed spikes in the early morning hours, and especially slow traffic on Friday afternoons.
+credit:
+# We want to thank you for curating this dataset! If you do not want a 
+# particular type of credit, please delete the related line.
+  post: Nicola Rennie
+  linkedin: https://www.linkedin.com/in/nicola-rennie
```

```suggestion
  bluesky: https://bsky.app/profile/nrennie.bsky.social
  linkedin: https://www.linkedin.com/in/nicola-rennie
```
I keep meaning to add Bluesky to the template! I assume this is ok to add, let me know if it isn't! Note: We don't actually use these (other than github) in the scripts yet, but I want them there so I can add them (hopefully fairly soon)!